diffusion-talent-accelerator/index.html

```diff
@@ -19,66 +19,19 @@
-                    </div>
-                    <div class="grid-col-12 desktop:grid-col-6 row1A">
-                        <img src="../assets/img/accelerator/row1.jpg" alt="Photo of woman speaking about the Diffusion Talent Accelerator (DTA)" />
-                    </div>
-                </div>
-            </div>       
-        </div>    
-    </section>
-    
-    <section id="secondary-row2" class="secondary-row two">
-        <div class="secondary-row__content-wrapper">
-            <div class="grid-container">
-                <div class="position-absolute row-custom-background"></div>
-                <div class="position-absolute row2-lines">
-                    <img src="../assets/img/accelerator/row2_lines.svg" alt="2 lines background image (blue and purple)" />
-                </div>
-                <div class="grid-row grid-gap">
-                    <div class="grid-col-12 desktop:grid-col-6 flex-align-self-center row2A">
-                        <img src="../assets/img/accelerator/row2.jpg" alt="Photo of a group of women and men collaborating and innovating within the VHA Innovation Ecosystem" />
-                    </div>
-                    <div class="grid-col-12 desktop:grid-col-6 flex-align-self-center row2B">
-                        <div class="grid-col-12 mobile-lg:padding-x-3 mobile:padding-x-1">
-                            <h3 class="margin-y-0">Empower Your Impact:</h3>
-                            <h2 class="margin-y-0">Collaborate and Innovate Within the VHA Innovation Ecosystem</h2>
-                            <p>VHA Innovation Ecosystem is seeking VISNs and National Program Offices that wish to dedicate their FTE to diffusion efforts, to strengthen their operations and expand VHA's human talent pool capable of replicating and scaling Promising Practices.</p>
-                        </div>
-                    </div>
-                </div>
-            </div>  
-            <div class="position-absolute row2-lines-mobile">
-                <img src="../assets/img/accelerator/row2_lines_mobile.svg" alt="2 lines background image (blue and purple)" />
-            </div>  
-        </div>    
-    </section>
-
-    <section id="secondary-banner" class="secondary-banner">
-        <div class="secondary-banner__content-wrapper">
-            <div class="grid-container">
-                <div class="grid-row grid-gap">
-                    <div class="grid-col-12">
-                        <div class="secondary-banner__wordmark text-center">
-                            <h2>Diffusion Talent Accelerator Cohort 1 is underway.</h2>
-                            <div class="secondary-banner__tagline">Diffusion of Excellence will be recruiting Cohort 2 in January of 2024. <br />For more information, please email <a href="mailto:VACODiffusionAccelerator@va.gov">VACODiffusionAccelerator@va.gov</a>.</div>
```

Comment:
The copy should be updated to : Recruitment is currently closed. Applications for the next cohort will open
in January 2025. For more information, please email
VHADiffusionTalentAccelerator@VA.gov. (Alec's doc for reference)